Add note about otel endpoint param being only the base url (#4432)
apollographql/router: docs/source/configuration/telemetry/exporters/tracing/otlp.mdx

* http://127.0.0.1:4317 for gRPC 
* http://127.0.0.1:4318 for HTTP

<Note>

> Specify only the base URL in the endpoint parameter. The `/v1/traces` portion of the URL will be automatically added to match the [Open Telemetry specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request).

Reviewer: ```suggestion
Specify only the base URL in the endpoint parameter. The router automatically adds the `/v1/traces` portion of the URL to match the [OpenTelemetry specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request).
```
